Cedrick

cedrick00 opened the pull request:
Re sized the tile images and fixed its padding
Fixed the side category and made a hover effect that shows all the subcategories
Fixed the padding of enter email address text box and button on mobile mode
so it aligns to center
Added search box for item
Added container for featured products
Added footer menus
Added footer

cedrick00 committed:
Added CSS elements for homepage UI

cedrick00 committed:
Fixed Image size and Paddings

cedrick00 committed:
Added Footer

cedrick00 committed:
Fixed columns of images for mobile view

cedrick00 committed:
Fixed showing of sub categories

cedrick00 committed:
Fixed image paddings and size to make it responsive

cedrick00 committed:
Change conflict CSS element name

cedrick00 committed:
Fixed image paddings and position for mobile view

cedrick00 committed:
Added shadow and radius in images

cedrick00 committed:
Fixed the image size and row and column alignment to make it centered

cedrick00 committed:
Added container for the featured products
Added menus in the footer
Added search bar for item
fixed paddings and margins for mobile and tablet view
resized images for small screens

cedrick00 committed:
Re-align social images and sign-up for newsletter textbox in left
Re-align Payment options images in left
Fixed border colors of tile so it doesn't look moving when hover
Hide some of the brands we offer images during mobile/tablet mode

jsdecena commented:
Almost there but there are still changes that needs to be implemented.
- review the width of the category ul
- take out the inline css's
- for the footer, make it as col-xs-6 instead col-xs-3

Please review thoroughly before making another pull request. Thanks!

cedrick00 committed:
Fixed the width of side category, so when category has a long
character, it will automatically fit in the container
Removed all inline css
Changed column size of footer menu to 6
